lib/manager/composer/artifacts.ts

```diff
@@ -110,7 +110,9 @@ export async function updateArtifacts({
     logger.debug('No composer.lock found');
     return null;
   }
-  await ensureLocalDir(getSiblingFileName(packageFileName, 'vendor'));
+
+  const vendorDir = getSiblingFileName(packageFileName, 'vendor');
+  await ensureLocalDir(vendorDir);
```

Comment:
I actually forget why we create the `vendor/` dir. It could possibly be for legacy reasons. Fine to leave like this for backwards-compatibility reasons.